scheds/rust/scx_rusty/src/load_balance.rs

```diff
@@ -879,9 +888,19 @@ impl<'a, 'b> LoadBalancer<'a, 'b> {
                     pull_node.domains.insert(pull_dom);
                     break;
                 }
-                let transferred = self.try_find_move_task((&mut push_dom, push_imbal),
+                let mut transferred = self.try_find_move_task((&mut push_dom, push_imbal),
```

Comment:
It might be a good idea to avoid doing this work altogether if `preferred_dom_mask` is zero, as that will probably be the common case (right?). But this is fine for now, we can address this in a follow-on change if we choose. Thanks!